Added Dotenv

charlie1404 opened the pull request:
Added Dotenv
Modified instruction to set up in README.md.

VarunDevPro commented:
This is really a good one, but will the cp command work on all the OS, if not try to add support for all the OS.

skyshader commented:
Looks good to me. Have you verified if things are working as expected? @charlie1404

VarunDevPro commented:
Ok Since I work only with Windows I don't know whether those commands work similarly on Mac and Linux. I have seen that copy is used instead of cp for Windows. 

I have also seen that some commands which are not recognised in **Command Prompt** can be recognised by **git bash** on Windows. 
If this is the case with **cp**, we should tell the users clearly when to use what.

VarunDevPro commented on server/server.js:
```diff
@@ -1,3 +1,8 @@
+const envVars = require("dotenv").config();
```
Yes it will good have simpler name.

VarunDevPro commented on server/.gitignore:
```diff
@@ -11,3 +11,8 @@ yarn-error.log*
 *.sln
+
+.env
+.env.example
```
Yes, users should know how to configure their env file so `env.example` should not be ignored.

charlie1404 commented:
I have added .env.example to gitignore but project is using env.example, as in linux or similar os'es it is hidden, but env.example will be visible in file explorer
though I have removed that from gitignore, and changed envVars to env only

skyshader commented:
@charlie1404 can you submit the PR to `develop` branch as we have moved our work on that. If I merge here, it might cause a lot of merge issues.

charlie1404 commented:
How do i Move this Pull request to develop branch, Please help

charlie1404 commented:
Closing this pull request in favor of pull #48
